src/briefcase/integrations/android_sdk.py

```diff
@@ -377,8 +377,8 @@ def select_target_device(self, device_or_avd):
             if avd:
                 # It's a running emulator
                 running_avds[avd] = d
-                full_name = "@{avd} ({name} emulator)".format(
-                    avd=avd, name=name,
+                full_name = "@{avd} (running emulator)".format(
```

Comment:
IMHO, this is a decrease in information, and I like `{name}` showing up b/c it lets me know if it's x86 vs. x86_64, etc. Up to you, though.